eva_cttv_pipeline/clinvar_to_evidence_strings.py

```diff
@@ -135,8 +135,8 @@ def write_output(self, dir_out):
         # Contains urls provided by Gary which are not yet included within EFO
         with utilities.open_file(dir_out + '/' + config.UNAVAILABLE_EFO_FILE_NAME, 'wt') as fdw:
             fdw.write('Trait\tCount\n')
-            for url in self.unavailable_efo_dict:
-                fdw.write(url.encode('utf8') + '\t' + str(self.unavailable_efo_dict[url]) + '\n')
+            for clinvar_name in list(self.unavailable_efo):
```

Comment:
For some reason I wrongfully remembered that you couldn't iterate over a set. Changed.